dll/apisets/api-ms-win-crt-process-l1-1-0.spec

```diff
@@ -11,7 +11,7 @@
 @ stdcall _execve() msvcrt._execve
 @ stdcall _execvp() msvcrt._execvp
 @ stdcall _execvpe() msvcrt._execvpe
-@ stdcall _loaddll() msvcrt._loaddll
+@ stdcall -version=0x502 -arch=i386,x86_64 _loaddll() msvcrt._loaddll
```

Comment:
From my point of view, `0x502` does not make sense for an apiset.

Reply:
Looking at wine specfile, -version and -arch does not exist so that was from the generator (after it checked for reactos functions), I could remove this two extra options.

Reply:
It probably inferred that from our exported _loaddll function:
`@ cdecl -arch=x86_64 -version=0x502 _loaddll(str)`